lib/payoneer/client.rb

```diff
@@ -106,5 +108,14 @@ def post(method_name, params = {})
         Response.new(Response::OK_STATUS_CODE, hash)
       end
     end
+
+    def create_response(hash)
```

Comment:
For some reason, if I try to use this helper method in the `post` method which is used for the normal payouts, I get some spec failures. I'm not quite sure why and I'm currently debugging this.

Reply:
Is it because of the private scope? Sorry if that was dumb 😛

Reply:
I wish that was the reason :(, would be an easier fix probably haha. It has to do something with what's returned from the methods that use the `post` method. For example `payee_signup_url` should only return the part of the response associated with `token` like `"https://payouts.sandbox.payoneer.com/partners/lp.aspx?token=fake-token"`. But instead, what is returned is `"Token" => "https://payouts.sandbox.payoneer.com/partners/lp.aspx?token=fake-token"`. I'm not really sure why refactoring part of the code into a helper function caused this.